ch_backup/logic/lock_manager.py

```diff
@@ -107,14 +122,18 @@ def _zk_flock(self) -> None:
             client = self._zk_client.__enter__()
             if not client.exists(self._process_zk_lockfile_path):
                 client.create(self._process_zk_lockfile_path, makepath=True)
-            self._zk_lock = client.Lock(self._process_zk_lockfile_path)
+            self._zk_lock = client.Lock(
+                self._process_zk_lockfile_path, identifier=self._lock_id
+            )
             try:
                 _ = self._zk_lock.acquire(blocking=True, timeout=self._lock_timeout)
-                logging.debug("Lock was acquired")
+                logging.debug("Lock was acquired.")
```

Comment:
Lets clarify that it is ZK lock.
If we really need it maybe we should add the same log info to flock?

But for me it seems we do not need this at all - in case of problem with lock we are getting error's message.

Reply:
Yes, probably redundant. I will remove it

Reply:
Do you think useful to include the zk_lock id in the error message?